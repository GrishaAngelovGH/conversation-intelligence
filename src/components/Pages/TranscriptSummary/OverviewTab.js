import Attendee from "../common/Attendee"

const OverviewTab = () => (
  <div className="row">
    <div className="col-md-12">
      <div className="row justify-content-between">
        <div className="col-md-5">
          <span className="fw-bold">Related Records</span>
          <div>
            <i className="bi bi-journal-check"></i>
            <i className="bi bi-bookmark ms-3"></i>
            <i className="bi bi-telephone-plus ms-3"></i>
            <i className="bi bi-inboxes ms-3"></i>
          </div>
        </div>
      </div>

      <div className="row mt-3 justify-content-between">
        <div className="col-md-12">
          <div className="fw-bold">Tags</div>
          <button className="btn btn-primary">Add call tags <i className="bi bi-plus-circle"></i></button>
        </div>
      </div>

      <div className="row mt-3 justify-content-between">
        <div className="col-md-12">
          <div className="fw-bold">Attendees</div>
          <Attendee shortName="JS" fullName="James Smith" />
          <Attendee shortName="IG" fullName="Isabel Garcia" />
          <Attendee shortName="MB" fullName="Madison Butler" />
        </div>
      </div>
    </div>
  </div>
)

export default OverviewTab