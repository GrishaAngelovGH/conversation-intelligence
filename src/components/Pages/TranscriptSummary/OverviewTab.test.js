import { render } from '@testing-library/react'

import OverviewTab from './OverviewTab'

test('should render component', () => {
  const container = render(<OverviewTab />)

  expect(container).toMatchSnapshot()
})