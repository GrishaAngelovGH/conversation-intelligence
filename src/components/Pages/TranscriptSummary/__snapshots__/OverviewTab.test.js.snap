// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="row justify-content-between"
          >
            <div
              class="col-md-5"
            >
              <span
                class="fw-bold"
              >
                Related Records
              </span>
              <div>
                <i
                  class="bi bi-journal-check"
                />
                <i
                  class="bi bi-bookmark ms-3"
                />
                <i
                  class="bi bi-telephone-plus ms-3"
                />
                <i
                  class="bi bi-inboxes ms-3"
                />
              </div>
            </div>
          </div>
          <div
            class="row mt-3 justify-content-between"
          >
            <div
              class="col-md-12"
            >
              <div
                class="fw-bold"
              >
                Tags
              </div>
              <button
                class="btn btn-primary"
              >
                Add call tags 
                <i
                  class="bi bi-plus-circle"
                />
              </button>
            </div>
          </div>
          <div
            class="row mt-3 justify-content-between"
          >
            <div
              class="col-md-12"
            >
              <div
                class="fw-bold"
              >
                Attendees
              </div>
              <div
                class="row mt-1 align-items-center"
              >
                <div
                  class="col-md-2"
                >
                  <div
                    class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                    style="width: 40px; height: 40px;"
                  >
                    JS
                  </div>
                </div>
                <div
                  class="col-md-6"
                >
                  <h4
                    class="m-0"
                  >
                    James Smith
                  </h4>
                </div>
              </div>
              <div
                class="row mt-1 align-items-center"
              >
                <div
                  class="col-md-2"
                >
                  <div
                    class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                    style="width: 40px; height: 40px;"
                  >
                    IG
                  </div>
                </div>
                <div
                  class="col-md-6"
                >
                  <h4
                    class="m-0"
                  >
                    Isabel Garcia
                  </h4>
                </div>
              </div>
              <div
                class="row mt-1 align-items-center"
              >
                <div
                  class="col-md-2"
                >
                  <div
                    class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                    style="width: 40px; height: 40px;"
                  >
                    MB
                  </div>
                </div>
                <div
                  class="col-md-6"
                >
                  <h4
                    class="m-0"
                  >
                    Madison Butler
                  </h4>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row"
    >
      <div
        class="col-md-12"
      >
        <div
          class="row justify-content-between"
        >
          <div
            class="col-md-5"
          >
            <span
              class="fw-bold"
            >
              Related Records
            </span>
            <div>
              <i
                class="bi bi-journal-check"
              />
              <i
                class="bi bi-bookmark ms-3"
              />
              <i
                class="bi bi-telephone-plus ms-3"
              />
              <i
                class="bi bi-inboxes ms-3"
              />
            </div>
          </div>
        </div>
        <div
          class="row mt-3 justify-content-between"
        >
          <div
            class="col-md-12"
          >
            <div
              class="fw-bold"
            >
              Tags
            </div>
            <button
              class="btn btn-primary"
            >
              Add call tags 
              <i
                class="bi bi-plus-circle"
              />
            </button>
          </div>
        </div>
        <div
          class="row mt-3 justify-content-between"
        >
          <div
            class="col-md-12"
          >
            <div
              class="fw-bold"
            >
              Attendees
            </div>
            <div
              class="row mt-1 align-items-center"
            >
              <div
                class="col-md-2"
              >
                <div
                  class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                  style="width: 40px; height: 40px;"
                >
                  JS
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <h4
                  class="m-0"
                >
                  James Smith
                </h4>
              </div>
            </div>
            <div
              class="row mt-1 align-items-center"
            >
              <div
                class="col-md-2"
              >
                <div
                  class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                  style="width: 40px; height: 40px;"
                >
                  IG
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <h4
                  class="m-0"
                >
                  Isabel Garcia
                </h4>
              </div>
            </div>
            <div
              class="row mt-1 align-items-center"
            >
              <div
                class="col-md-2"
              >
                <div
                  class="rounded-circle bg-primary d-flex justify-content-center align-items-center text-white fw-bold"
                  style="width: 40px; height: 40px;"
                >
                  MB
                </div>
              </div>
              <div
                class="col-md-6"
              >
                <h4
                  class="m-0"
                >
                  Madison Butler
                </h4>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
